src/App.jsx

```diff
@@ -14,57 +15,64 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGoods, setSelectedGoods] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  const clearClick = () => {
```

Comment:
As far as I remember, callback functions for buttons should start with on or handle.
Example: onClickClear or handleClickClear

Reply:
agreed